src/index.js

```diff
@@ -1,15 +1,15 @@
 const candidateSelectors = [
-  'input',
-  'select',
-  'textarea',
-  'a[href]',
-  'button',
-  '[tabindex]:not(slot)',
-  'audio[controls]',
-  'video[controls]',
-  '[contenteditable]:not([contenteditable="false"])',
-  'details>summary:first-of-type',
-  'details',
+  'input:not([inert] *)',
```

Comment:
you can add `:not([inert])` to all of these selectors to save the extra JavaScript check on the element itself

Reply:
Thanks for pointing this out. We can eliminate more candidates right off the bat with the addition of `:not([inert])`, which I have done. I didn't use the multi-selector syntax since the exhaustive syntax has broader support (just in case...).